.circleci/config.yml

```diff
@@ -16,7 +16,502 @@ aliases:
     branches:
       only: master
 
+executors:
+  grafana-build:
+    docker:
+      - image: grafana/build-container:1.2.13
+  grafana-publish:
+    docker:
+      - image: grafana/grafana-ci-deploy:1.2.3
+  docker:
+    machine:
+      image: circleci/classic:201808-01
+
 jobs:
+  install-grabpl:
+    description: Install the Grafana Build Pipeline tool
+    executor: grafana-build
+    steps:
+      - run: |
+          echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | \
+            tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+          curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+          apt-get update && apt-get install -f google-cloud-sdk
+          echo $GCP_GRAFANA_UPLOAD_KEY > /tmp/gcpkey.json
+          gsutil cp gs://grafana-build-pipeline/grabpl bin/grabpl
```

Comment:
how do we version this so that a version of the build pipeline version is in sync with grafana branch/version?  Always taking the latest build pipeline seems it will cause issues when we want to build older branches

Reply:
@torkelo Agree - my thinking so far is that we version the build pipeline tool and tie Grafana branches to a compatible version.

Reply:
To specify, my thinking is we can come up with a version scheme later, when we're past prototyping.

Reply:
Why don't use go get? The github token should be able to access that repo. To be able to use it locally would make sense to not have to download tooling from gcp.